### DISCUSSION TITLE
Run CI called “sqlalchemy_dbapi_main”

### DISCUSSION THREAD

**Discussion by Kajalagg98** [sqlalchemy/sqlalchemy#7414] [Usage Questions]
How we can run CI “sqlalchemy_dbapi_main” against cx_Oracle?

**Comment by CaselIT** [comment]
Hi,

I don't know what `sqlalchemy_dbapi_main` is. The documentation on how to run the tests is in the file named `README.unittests.rst`

**Reply by zzzeek** [reply]
It's one of our Jenkins jobs.   It should be doing cx_oracle.

**Reply by Kajalagg98** [reply]
Got it. Thanks.
